Documentation/compatibility/SignedXml.GetPublicKey-returns-RSACng-on-net462.md

```diff
@@ -0,0 +1,28 @@
+## SignedXml.GetPublicKey returns RSACng on net462 (or lightup) without retargeting change
+
+### Scope
+Edge
+
+### Version Introduced
+4.6.2
+
+### Version Reverted
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+We let an object return type change in net462 without quirking it. This change adds a quirk to use the old method of getting the return value (certificate.PublicKey.Key) instead of the new method (certificate.GetAnyPublicKey)
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+To use the old method of getting a key, set the switch: Switch.System.Security.Cryptography.Xml.SignedXmlUseLegacyCertificatePrivateKey
```

Comment:
Since the change description is unclear, it's also unclear what the switch does. Concrete detail would be very helpful in this topic.

Reply:
I'll add some more details.